plugins/Login/Login.php

```diff
@@ -225,7 +225,7 @@ public function noAccess(Exception $exception)
      * Set login name and authentication token for API request.
      * Listens to API.Request.authenticate hook.
      */
-    public function ApiRequestAuthenticate($tokenAuth)
+    public function apiRequestAuthenticate($tokenAuth)
```

Comment:
This is BC I'd say since it's a public method, but I guess we could be clearer on what constitutes public API for core and what is and isn't covered by semver.

Reply:
It actually only changes the writing case of the method name. PHP still shouldn't make a difference there. So calling `apiRequestAuthenticate` or `ApiRequestAuthenticate` ends up being the same. It's not even possible to define the method twice with different writing. And in terms of extending such a class it's actually the same. See e.g. https://3v4l.org/pO66N

Reply:
Cool, that's good to know, thanks for confirming that.